### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by choznerol**

### CODE REVIEW

**Comment by reinink** [comment]
Thanks for catching this. I'm thinking "for" might be even better than "within". Do mind updating your PR? 🙏

File: pages/shared-data.mdx [hunk 1]
```diff
@@ -20,7 +20,7 @@ Sometimes need to access certain data on numerous pages within your application.
 
 ## Sharing data
 
-The server-side adapters provide a way to preassign shared data which each request. This is typically done outside of your controllers. Shared data will be automatically merged with the page props provided in your controller.
+The server-side adapters provide a way to preassign shared data within each request. This is typically done outside of your controllers. Shared data will be automatically merged with the page props provided in your controller.
```
Comment (choznerol) [inline pages/shared-data.mdx]:
```suggestion
The server-side adapters provide a way to preassign shared data for each request. This is typically done outside of your controllers. Shared data will be automatically merged with the page props provided in your controller.
```

**Comment by choznerol** [comment]
Updated :)

**Comment by reinink** [comment]
Thanks! 👍

**Pull request merged**